t/120-re-find.t

```diff
@@ -917,3 +917,35 @@ to: 4
 pos: 5
 --- no_error_log
 [error]
+
+
+
+=== TEST 32: ignore match limit in DFA mode
+--- http_config
+    lua_regex_match_limit 1;
+--- config
+    location /re {
+        content_by_lua '
```

Comment:
we should use `content_by_lua_block` in new tests.

Reply:
done